drivers/clock_control/clock_control_esp32.c

```diff
@@ -41,6 +41,9 @@
 #include <esp32c6/rom/rtc.h>
 #include <soc/dport_access.h>
 #include <hal/clk_tree_ll.h>
+#include <hal/usb_serial_jtag_ll.h>
+#include <esp_private/esp_pmu.h>
+#include <ocode_init.h>
 #endif /* CONFIG_SOC_SERIES_ESP32xx */
```

Comment:
compliance check usually dont like if #endif comments differs from the #if statement